File: remove_cs.py
import os
import glob

base_dir = r"c:/Users/dines/OneDrive/Desktop/feb project/PURPLE/"
for filepath in glob.glob(os.path.join(base_dir, '**', '*.html'), recursive=True):
    try:
        with open(filepath, 'r', encoding='utf-8') as f:
            lines = f.readlines()
        
        new_lines = []
        changed = False
        for line in lines:
            # Delete nav link to customer service
            if '>Customer Service</a></li>' in line and 'customer-service.html' in line:
                changed = True
                continue
            
            # Change cta in services.html
            if 'href="customer-service.html#"' in line and 'class="service-cta"' in line:
                new_lines.append(line.replace('customer-service.html#', 'contact.html#'))
                changed = True
                continue
            
            # Change button in dashboard
            if 'href="customer-service.html"' in line and 'class="dash-btn' in line:
                new_lines.append(line.replace('customer-service.html', 'contact.html'))
                changed = True
                continue
                
            new_lines.append(line)
        
        if changed:
            with open(filepath, 'w', encoding='utf-8') as f:
                f.writelines(new_lines)
            print(f"Updated: {filepath}")
    except Exception as e:
        print(f"Error processing {filepath}: {e}")

File: remove_cs.js
const fs = require('fs');
const path = require('path');

function walkDir(dir, callback) {
    fs.readdirSync(dir).forEach(f => {
        let dirPath = path.join(dir, f);
        let isDirectory = fs.statSync(dirPath).isDirectory();
        if (isDirectory) {
            walkDir(dirPath, callback);
        } else {
            callback(path.join(dir, f));
        }
    });
}

const baseDir = "c:/Users/dines/OneDrive/Desktop/feb project/PURPLE";

walkDir(baseDir, (filepath) => {
    if (!filepath.endsWith('.html')) return;

    try {
        let content = fs.readFileSync(filepath, 'utf-8');
        let lines = content.split('\n');
        let changed = false;

        let new_lines = lines.filter(line => {
            // Delete nav link to customer service
            if (line.includes('>Customer Service</a></li>') && line.includes('customer-service.html')) {
                changed = true;
                return false;
            }
            return true;
        }).map(line => {
            // Change cta in services.html
            if (line.includes('href="customer-service.html#"') && line.includes('class="service-cta"')) {
                changed = true;
                return line.replace('customer-service.html#', 'contact.html#');
            }

            // Change button in dashboard
            if (line.includes('href="customer-service.html"') && line.includes('class="dash-btn')) {
                changed = true;
                return line.replace('customer-service.html', 'contact.html');
            }
            return line;
        });

        if (changed) {
            fs.writeFileSync(filepath, new_lines.join('\n'), 'utf-8');
            console.log("Updated: " + filepath);
        }
    } catch (e) {
        console.error("Error processing " + filepath + ": " + e);
    }
});
